### PR TITLE
MVRF Update

### PR DESCRIPTION
**Pull request by Kuroitsune**
Future setup for Moved Reference in TES3.

### CODE REVIEW

**Comment by Kuroitsune** [comment]
Removed all my comments to make the def file cleaner looking. All of it was transferred to a comment on the previous pull request.

Updates:
[TES3] INFO\SCVR - Fixed incorrect hex data for the Global function in the enumerator.
[TES3] Added default values for DELE and other esoteric signatures found throughout TES3.

**Comment by Kuroitsune** [comment]
Forgot to put this in the pull request thingy. Last and final update for TES3.

[TES3] SSCR\DATA - Changed name to Numerical ID.
[TES3] SKIL\SKDT - Simplified the action Union by getting rid of it and using an array.
[TES3] SKIL\INDX - Renamed to Name.
[TES3] WEAP\WPDT - Restructured the Damage Type values.
[TES3] CREA,NPC_\NPDT - Marked Unknown flags as Junk flags.
[TES3] INGR\IRDT - Added special enumerators for Skills and Attributes to denote 'None' for special circumstances.
[TES3] LEVI\DATA - Fixed spelling error.
[TES3] REGN\WEAT - Removed the union in favor for a properly constructed struct.
[TES3] REGN\SNAM - Flipped the integer for Chance to signed to reflect the Construction Set.
[TES3] MVRF,REFR\NAM9 - Was misnamed as Gold Value, it is the Count for the object.
[TES3] MGEF\INDX - Renamed to Effect.
[TES3] SSCR - Only the Editor ID is displayed in the object list window.
[TES3] SOUN - Only the Editor ID is displayed in the object list window.
[TES3] ALL - Floats are now limited to the number of digits available in the Construction Set.

File: Core/wbDefinitionsTES3.pas [hunk 1]
```diff
@@ -1542,77 +1444,110 @@ procedure DefineTES3;
...
+    )
   ]).SetFormIDBase($40);
 
-  //Standardized
   wbRecord(LEVI, 'Leveled Item', [
     wbString(NAME, 'Editor ID'),
-    wbInteger(DELE, 'Deleted', itU32, wbEnum(['Deleted'],[])),
-    wbInteger(DATA, 'Levlved Flags', itU32, wbLeveledFlags),
+    wbInteger(DELE, 'Deleted', itU32, wbEnum(['Deleted'],[])).SetDefaultNativeValue(0),
+    wbInteger(DATA, 'Levelved Flags', itU32, wbLeveledFlags),
```
Comment (C3pa) [inline Core/wbDefinitionsTES3.pas:1529]: `Leveled`?

**Comment by Kuroitsune** [comment]
I fixed the spelling error,... only to make another one.

It will get fixed in my next patch of stuff or if Rob gets to it first.

**Pull request merged**